examples/v3.0/petstore-expanded.yaml

```
@@ -12,7 +12,7 @@ info:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
 servers:
-  - url: http://petstore.swagger.io/api
+  - url: https://petstore.swagger.io
```

```suggestion
  - url: https://petstore.swagger.io/v2
```

I believe this is required too, though I'm not sure what endpoints if any have changed between v1 and v2 of the API. Ping @hkosova

---

`petstore.swagger.io` API has and always had `/v2` only.

There's a newer version at https://petstore3.swagger.io/, but I don't recall if or how it's different (apart from using the OAS3 format).

ping @ponelat